pkg/providers/ovirt/validation/validators/vm-validator.go

```diff
@@ -93,6 +96,22 @@ func isValidBios(vm *ovirtsdk.Vm) []ValidationFailure {
 	return results
 }
 
+func isValidStatus(vm *ovirtsdk.Vm) (ValidationFailure, bool) {
+	if status, ok := vm.Status(); ok {
+		if status != ovirtsdk.VMSTATUS_UP && status != ovirtsdk.VMSTATUS_DOWN {
+			return ValidationFailure{
+				ID:      VMStatusID,
+				Message: fmt.Sprintf("VM has illegal status: %v", status),
```

Comment:
Can you please write here, what the valid statuses are? So user know.

Reply:
Good point. Thanks!

Reply:
Done.